controllers/endPoint.controller.js

```diff
@@ -0,0 +1,140 @@
+
+exports.getEndPoint = (req, res, next) => {
+    const endpoints = {
```

Comment:
Ewww! We don't need all this logic in the controller. You should have an endpoints.json file with this information in, and then you can require it in at the top of this file to use the data.